mm2src/coins/tendermint/tendermint_tx_history_v2.rs

```diff
@@ -715,14 +732,14 @@ where
                     },
                     token_id,
                 },
-                (TransferEventType::IBCSend, _) | (TransferEventType::IBCReceive, _) => {
-                    TransactionType::TendermintIBCTransfer
+                (TransferEventType::IBCSend, token_id) | (TransferEventType::IBCReceive, token_id) => {
+                    TransactionType::TendermintIBCTransfer { token_id }
                 },
                 (TransferEventType::Delegate, _) => TransactionType::StakingDelegation,
                 (TransferEventType::Undelegate, _) => TransactionType::RemoveDelegation,
-                (_, Some(token_id)) => TransactionType::TokenTransfer(token_id),
                 (TransferEventType::ClaimRewards, _) => TransactionType::ClaimDelegationRewards,
-                _ => TransactionType::StandardTransfer,
+                (_, Some(token_id)) => TransactionType::TokenTransfer(token_id),
+                (_, None) => TransactionType::StandardTransfer,
```

Comment:
what about replacing the `_` in the match arm with `TransferEventType::Standard` as it's the only remaining one? right?

Reply:
Yeah, fixed.